dist/index.html

```diff
@@ -21,7 +21,7 @@
 			</div>
 		</header>
 		<main id="site-main" class="main-content">
-			<div class="container shim">
+			<div class="container shim js-api" data-api="../../api/doctors.json">
```

Comment:
[comment/suggestion] I think you could still stand to work on your class names. Really give some consideration to whether the names you're giving to things have any contextual meaning to other people who pick up your project (i.e., I don't know what `container` or `shim` actually mean or what they do)